docs/src/main/sphinx/release/release-469.md

```diff
@@ -0,0 +1,141 @@
+
+## Delta Lake connector
+
+* Add support for SSE-C in S3 security mapping. ({issue}`24566`)
+* Allow configuring the endpoint for the Google Storage file system with the
+  `gcs.endpoint` property. ({issue}`24626`)
+* Improve performance of reading from new Delta Lake table data by compressing
+  files with `ZSTD` by default, instead of the previous `SNAPPY`.
+  ({issue}`17426`)
+* Improve performance of queries on tables with large transaction log JSON
+  files. ({issue}`24491`)
+* Improve performance of reading from Parquet files with a large number of row
+  groups. ({issue}`24618`)
+* Improve performance for the `OPTIMIZE` statement by enabling concurrent
+  execution. ({issue}`16985`)
+* Improve performance of reading from large files on S3. ({issue}`24521`)
+* Fix JMX metrics for file system caching with multiple catalogs. ({issue}`24510`)
+* Fix reading from tables when using the Alluxio file system. ({issue}`23815`)
+* Fix incorrect results when writing with deletion vectors. ({issue}`24648`)
+* Fix incorrect results when reading deletion vectors. ({issue}`22972`)
+
+## Faker connector
+
+* Add support for views. ({issue}`24242`)
+* Support generating sequences. ({issue}`24590`)
+* {{breaking}} Replace predicate pushdown with the `min`, `max`, and `options`
+  column properties. ({issue}`24147`)
+
+## Hive connector
+
+* Add support for SSE-C in S3 security mapping. ({issue}`24566`)
+* Allow configuring the endpoint for the Google Storage file system with the
+  `gcs.endpoint` property. ({issue}`24626`)
+* Improve performance of reading from Parquet files with a large number of row
+  groups. ({issue}`24618`)
+* Improve performance of reading from large files on S3. ({issue}`24521`)
+* Fix reporting of AWS SDK client retry count metrics to report both client
+  level logical retries and lower level HTTP client retries separately. ({issue}`24606`)
+* Fix JMX metrics for file system caching with multiple catalogs. ({issue}`24510`)
+* Fix reading from tables when using the Alluxio file system. ({issue}`23815`)
+* Prevent writing of invalid data for NaN, Infinity, -Infinity values to JSON
+  files. ({issue}`24558`)
+
+## Hudi connector
+
+* Add support for SSE-C in S3 security mapping. ({issue}`24566`)
+* Allow configuring the endpoint for the Google Storage file system with the
+  `gcs.endpoint` property. ({issue}`24626`)
+* Improve performance of reading from Parquet files with a large number of row
+  groups. ({issue}`24618`)
+* Improve performance of reading from large files on S3. ({issue}`24521`)
+
+## Iceberg connector
+
+* Add support for SSE-C in S3 security mapping. ({issue}`24566`)
+* Allow configuring the endpoint for the Google Storage file system with the
+  `gcs.endpoint` property. ({issue}`24626`)
+* Add `$entries` metadata table. ({issue}`24172`)
+* Add `$all_entries` metadata table. ({issue}`24543`)
+* Add `system.iceberg_tables` table function to allow listing only Iceberg
+  tables. ({issue}`24469`)
```

Comment:
We should remove this entry as we decided to revert the change. #24719

Reply:
Done